package.json

```diff
@@ -23,12 +23,13 @@
     "npm": "^6.0.0"
   },
   "devDependencies": {
-    "babel-core": "^6.26.3",
+    "@babel/core": "^7.3.4",
+    "@babel/preset-env": "^7.3.4",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
babel-loader internally seems to be using babel-core, even though it's not specified as a dependency. In any case, this "bridge" version uses `@babel/core`, so we only really have v7 installed.